src/main/resources/docs/description/description.json

```diff
@@ -116,7 +120,14 @@
 }, {
   "patternId" : "Drupal_Files_LineLength",
   "title" : "Files: Line Length",
-  "description" : "Checks comment lines in the file, and throws warnings if they are over 80\ncharacters in length."
+  "description" : "Checks comment lines in the file, and throws warnings if they are over 80\ncharacters in length.",
```

Comment:
Can you substitute `\n`s with a spaces?

Reply:
Nice catch! 🙏 
Doing it in another PR https://github.com/codacy/codacy-codesniffer/pull/87